docs/docs/api/promise.on.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: api
+id: promise.on
+title: Promise.on
+---
+
+
+[← Back To API Reference](/docs/api-reference.html)
+<div class="api-code-section"><markdown>
+##Promise.on
+
+```js
+Promise.on(String eventName("created"|"chained"|"fulfilled"|"rejected"|"following"|"cancelled"), Function handler, [Boolean simpleEventAPI]) -> undefined
```

Comment:
"following" is internal terminology, it should probably be "resolved". Events should probably be on `process` or `window` so that they can be tracked using one set of listeners for all bluebird instances.